docs/Creator-V2-Whats-New.md

```diff
@@ -1,309 +1,195 @@
+This new internal design also prompted us to extend the Library with new features that were demanded by the Survey Creator. For example, question titles can now contain [context actions](https://surveyjs.io/Examples/Library?id=survey-titleactions), and [Comment](https://surveyjs.io/Documentation/Library?id=questioncommentmodel) questions can [automatically grow or shrink](https://surveyjs.io/Documentation/Library?id=questioncommentmodel#autoGrow) the input field to accommodate the content.
 
-In Creator V1, we already used UI elements of SurveyJS Library but in just two places: within the Designer tab (on the design surface) and within the Test tab (where an instance of a survey widget is displayed for testing purposes).
+Since Survey Creator UI elements are surveys, you can customize them as you would customize surveys. For example, the Property Grid is a one-page survey in which every property is a question. To introduce a new or override an existing property editor, you need to define a custom question JSON configuration and implement functions that survey events call internally.
 
-In Creator V2, we have an improved situation. SurveyJS Library elements are now used in a more extensive way - almost everywhere within a Creator UI.
+[View Custom Property Editor example](https://surveyjs.io/Examples/Survey-Creator?id=custompropertyeditor&platform=ReactjsV2#content-js)
 
+### Lazy Rendering
 
-<a id="changes-in-surveyjs-library"></a>
+To improve performance on large multi-page surveys, we introduce support for Lazy Rendering to our SurveyJS Library. This feature allows the survey to render questions outside the viewport as skeletons instead of rendering them in full.
 
-### Changes in SurveyJS Library
-Some evolutionary changes in SurveyJS Library are listed below. 
+[View Lazy Rendering example](https://surveyjs.io/Examples/Library?id=survey-lazy)
 
+### Alternative Rendering for Adorners
 
-#### Performance improved for multi-page surveys
-Previously, SurveyJS Library could render up to 100 questions on one page with a good performance. However, when we started to implement support for Microsoft Word page layout in Creator V2 and tried to display all pages at once, we began to encounter performance issues. For instance, it was a problem to render 40 pages at once if each page contained 25 questions (a total of 1000 questions to render).  
+Previously, we added [adorners](https://surveyjs.io/Documentation/Survey-Creator?id=survey-customization#specify-adorner-availability) to survey elements _after_ the elements were rendered. This wasn't a very effective solution because we modified the DOM twice.
```

Comment:
```suggestion
Previously, we used to render elements first, and only then added [adorners](https://surveyjs.io/Documentation/Survey-Creator?id=survey-customization#specify-adorner-availability). This wasn't a very effective solution because we modified the DOM twice.
```